File: Array/1752. Check if Array Is Sorted and Rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) 
    {
        int n = nums.size();
        vector<int>t(nums.begin(),nums.end());
        sort(t.begin(),t.end());
        for(int i=0;i<n;i++)
        {
            nums.push_back(nums[0]);
            nums.erase(nums.begin());
            if(nums == t) return true;
        }
        return false;
    }
};

File: 1752. Check if Array Is Sorted and Rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        flag = False
        n = len(nums)
        for i in range(1, n):
            if nums[i] < nums[i - 1]:
                arr = nums[i:] + nums[:i]
                nums.sort()
                return nums == arr
        return True